### PR TITLE
added @fideloper

### PR DESCRIPTION
**Pull request by fideloper**
Adding myself, which almost feels grimey but hey - why not!?

### CODE REVIEW

**Comment by the-dijkstra** [comment]
A true gem.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 - **[@nunomaduro](https://www.youtube.com/@nunomaduro)**: Nuno Maduro ‧ Laravel team member
 - **[@LaravelDaily](https://www.youtube.com/@LaravelDaily)**: Povilas Korop ‧ Founder of Laravel Daily
 - **[@Codecourse](https://www.youtube.com/@codecourse)**: Alex Garrett-Smith ‧ Founder of Codecourse
-
+- **[@fideloper](https://www.youtube.com/@fideloper)**: Chris Fidao ‧ PHP and Servers
+-
```
Comment (Jubeki) [inline README.md:7]:
```suggestion

```

**Review by Jubeki — COMMENTED** [review]
Seems like you added another line with a dash.

**Comment by fideloper** [comment]
thanks on both counts!

**Review by the-dijkstra — APPROVED** [review]

**Pull request merged**